libV2/tasks/DiskUsage.js

```diff
@@ -46,6 +47,64 @@ class MonitorDiskUsage extends BaseTask {
             );
     }
 
+    // eslint-disable-next-line class-methods-use-this
+    _registerMetricHandlers() {
+        const isLocked = new promClient.Gauge({
+            name: 'utapi_monitor_disk_usage_is_locked',
+            help: 'Indicates whether the monitored warp 10 has had writes disabled',
+            labelNames: ['origin', 'containerName'],
+        });
+
+        const diskUsage = new promClient.Gauge({
+            name: 'utapi_monitor_disk_usage_bytes',
+            help: 'Total bytes used by warp 10',
+            labelNames: ['origin', 'containerName'],
+        });
+
+        const hardLimitRatio = new promClient.Gauge({
```

Comment:
Instead of storing the ratio in Prometheus, it might be better to store the hard limit value itself, so likely it will be constant until it gets changed, to follow the philosophy of storing the raw values instead of pre-computed ones. The ratio can be simply computed with a query doing the division with the disk space used, and it allows other types of queries like how much space remains before hitting the hard limit. It might be useful also to know if/when the hard limit was changed, which will be obvious when storing the hard limit value itself.